resources/web/style/xpack.pcss

```diff
@@ -19,17 +19,17 @@
    * inside the title. That "phrase" would be rendered in a funny way
    * without this.
    */
-  h1 > span.xpack,
-  h2 > span.xpack,
-  h3 > span.xpack,
-  h4 > span.xpack,
-  h5 > span.xpack,
-  h6 > span.xpack {
+  h1 > .xpack,
+  h2 > .xpack,
+  h3 > .xpack,
+  h4 > .xpack,
+  h5 > .xpack,
+  h6 > .xpack {
```

Comment:
Do we need the h1 specifiers as well? is `.xpack`

Reply:
*Probably*. It is difficult to say without digging around quite a bit to make sure the `xpack` class only exists there. We probably could clean it up if we hacked on the `xslt` some but I really try to do that as little as possible.